### ISSUE TITLE
The Granite model failed to use tool calling

### ISSUE THREAD

**Issue by Finenyaco** [gpustack/llama-box#17]
**Test Steps:**

1. Deploy model `QuantFactory/granite-3.1-8b-instruct-GGUF-Q8_0` from Hugging Face
2. Test the toolcalling feature using the chat.sh script.
```
LOG_FILE=~/Downloads/chat.log TOOLS_WITH=true ./chat.sh "What's the weather like today in ShenZhen?"
```

**Test Result:**

No correct answer.
```
(base)  ~/llama-box/llama-box/tools/ [main] LOG_FILE=~/Downloads/chat.log TOOLS_WITH=true ./chat.sh "What's the weather like today in ShenZhen?"
=====================================================
LOG_FILE          : /Users/sophia/Downloads/chat.log
API_URL           : http://192.168.50.174:40867
FREQUENCY_PENALTY : 0.0
LOGPROBS          : false
TOP_LOGPROBS      : null
MAX_TOKENS        : null
PRESENCE_PENALTY  : 0.0
RESPONSE_FORMAT   : text
SEED              : null
STOP              : null
TEMP              : 1
TOP_P             : 1
TOOLS             : ["get_temperature","get_weather","get_location"]
=====================================================

> What's the weather like today in ShenZhen?
<tool_call>[{"name": "get_weather", "arguments": {"location": "ShenZhen"}}
------------------------
- TTFT :      27.75ms  -
- TBT  :      11.01ms  -
- TPS  :      90.86    -
- TC   :       1.00s   -
------------------------
```

**Additional context**

- llama-box logs:
```
2024-12-24T12:54:31+08:00 - gpustack.worker.backends.base - DEBUG - Model file size: 8898168928
2024-12-24T12:54:31+08:00 - gpustack.worker.tools_manager - DEBUG - llama-box_v0.0.97 already exists, skipping download
2024-12-24T12:54:32+08:00 - gpustack.worker.downloaders - INFO - Downloading model QuantFactory/granite-3.1-8b-instruct-GGUF/granite-3.1-8b-instruct.Q8_0.gguf
Fetching 1 files: 100%|██████████| 1/1 [00:01<00:00,  1.14s/it]
2024-12-24T12:54:33+08:00 - gpustack.worker.downloaders - INFO - Downloaded model QuantFactory/granite-3.1-8b-instruct-GGUF/granite-3.1-8b-instruct.Q8_0.gguf
2024-12-24T12:54:33+08:00 - gpustack.worker.backends.base - DEBUG - __init__ execution time: 3.5022544860839844 seconds
2024-12-24T12:54:33+08:00 - gpustack.worker.backends.llama_box - INFO - Starting llama-box server
2024-12-24T12:54:33+08:00 - gpustack.worker.backends.llama_box - DEBUG - Run llama-box: /Users/sophia/data/gpustack/bin/llama-box_v0.0.97 with arguments: --host 0.0.0.0 --embeddings --gpu-layers 41 --parallel 4 --ctx-size 8192 --port 40867 --model /opt/models/huggingface/models--QuantFactory--granite-3.1-8b-instruct-GGUF/snapshots/ecfd779de05fafb1d987383374ef141d1515b9c2/./granite-3.1-8b-instruct.Q8_0.gguf --no-mmap --no-warmup
0.00.017.648 I ggml_cuda_init: GGML_CUDA_FORCE_MMQ:    no
0.00.017.649 I ggml_cuda_init: GGML_CUDA_FORCE_CUBLAS: no
0.00.017.649 I ggml_cuda_init: found 1 CUDA devices:
0.00.019.420 I   Device 0: NVIDIA GeForce RTX 4090 D, compute capability 8.9, VMM: yes
0.00.022.203 I 
0.00.022.205 I version: v0.0.97 (5a1a639)
0.00.022.205 I compiler: cc (GCC) 9.3.1 20200408 (Red Hat 9.3.1-2)
0.00.022.205 I target: x86_64-redhat-linux
0.00.022.205 I vendor: 
0.00.022.206 I   - llama.cpp b92a14a8 (4381)
0.00.022.206 I   - stable-diffusion.cpp c8837bd (208)
0.00.022.236 I system_info: n_threads = 6 (n_threads_batch = 6) / 20 | CUDA : ARCHS = 600,610,700,750,800,860,890,900 | F16 = 1 | USE_GRAPHS = 1 | PEER_MAX_BATCH_SIZE = 128 | CPU : SSE3 = 1 | SSSE3 = 1 | AVX = 1 | AVX2 = 1 | F16C = 1 | FMA = 1 | LLAMAFILE = 1 | AARCH64_REPACK = 1 | 
0.00.022.237 I 
0.00.022.291 I srv                      main: listening, hostname = 0.0.0.0, port = 40867, n_threads = 6 + 2
0.00.023.424 I srv                      main: loading model
0.00.023.425 I srv                load_model: loading model ''
0.00.141.001 I llama_load_model_from_file: using device CUDA0 (NVIDIA GeForce RTX 4090 D) - 23397 MiB free
0.00.146.746 I llama_model_loader: loaded meta data with 38 key-value pairs and 363 tensors from /opt/models/huggingface/models--QuantFactory--granite-3.1-8b-instruct-GGUF/snapshots/ecfd779de05fafb1d987383374ef141d1515b9c2/./granite-3.1-8b-instruct.Q8_0.gguf (version GGUF V3 (latest))
0.00.146.752 I llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.
0.00.146.754 I llama_model_loader: - kv   0:                       general.architecture str              = granite
0.00.146.755 I llama_model_loader: - kv   1:                               general.type str              = model
0.00.146.756 I llama_model_loader: - kv   2:                               general.name str              = Models
0.00.146.756 I llama_model_loader: - kv   3:                         general.size_label str              = 8.4B
0.00.146.756 I llama_model_loader: - kv   4:                            general.license str              = apache-2.0
0.00.146.757 I llama_model_loader: - kv   5:                   general.base_model.count u32              = 1
0.00.146.758 I llama_model_loader: - kv   6:                  general.base_model.0.name str              = Granite 3.1 8b Base
0.00.146.758 I llama_model_loader: - kv   7:          general.base_model.0.organization str              = Ibm Granite
0.00.146.759 I llama_model_loader: - kv   8:              general.base_model.0.repo_url str              = https://huggingface.co/ibm-granite/gr...
0.00.146.764 I llama_model_loader: - kv   9:                               general.tags arr[str,3]       = ["language", "granite-3.1", "text-gen...
0.00.146.765 I llama_model_loader: - kv  10:                        granite.block_count u32              = 40
0.00.146.765 I llama_model_loader: - kv  11:                     granite.context_length u32              = 131072
0.00.146.766 I llama_model_loader: - kv  12:                   granite.embedding_length u32              = 4096
0.00.146.766 I llama_model_loader: - kv  13:                granite.feed_forward_length u32              = 12800
0.00.146.766 I llama_model_loader: - kv  14:               granite.attention.head_count u32              = 32
0.00.146.767 I llama_model_loader: - kv  15:            granite.attention.head_count_kv u32              = 8
0.00.146.770 I llama_model_loader: - kv  16:                     granite.rope.freq_base f32              = 10000000.000000
0.00.146.771 I llama_model_loader: - kv  17:   granite.attention.layer_norm_rms_epsilon f32              = 0.000010
0.00.146.771 I llama_model_loader: - kv  18:                          general.file_type u32              = 7
0.00.146.771 I llama_model_loader: - kv  19:                         granite.vocab_size u32              = 49155
0.00.146.772 I llama_model_loader: - kv  20:               granite.rope.dimension_count u32              = 128
0.00.146.772 I llama_model_loader: - kv  21:                    granite.attention.scale f32              = 0.007812
0.00.146.773 I llama_model_loader: - kv  22:                    granite.embedding_scale f32              = 12.000000
0.00.146.773 I llama_model_loader: - kv  23:                     granite.residual_scale f32              = 0.220000
0.00.146.774 I llama_model_loader: - kv  24:                        granite.logit_scale f32              = 16.000000
0.00.146.774 I llama_model_loader: - kv  25:                       tokenizer.ggml.model str              = gpt2
0.00.146.775 I llama_model_loader: - kv  26:                         tokenizer.ggml.pre str              = refact
0.00.153.401 I llama_model_loader: - kv  27:                      tokenizer.ggml.tokens arr[str,49155]   = ["<|end_of_text|>", "<fim_prefix>", "...
0.00.155.459 I llama_model_loader: - kv  28:                  tokenizer.ggml.token_type arr[i32,49155]   = [3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, ...
0.00.161.691 I llama_model_loader: - kv  29:                      tokenizer.ggml.merges arr[str,48891]   = ["Ġ Ġ", "ĠĠ ĠĠ", "ĠĠĠĠ ĠĠ...
0.00.161.692 I llama_model_loader: - kv  30:                tokenizer.ggml.bos_token_id u32              = 0
0.00.161.692 I llama_model_loader: - kv  31:                tokenizer.ggml.eos_token_id u32              = 0
0.00.161.692 I llama_model_loader: - kv  32:            tokenizer.ggml.unknown_token_id u32              = 0
0.00.161.693 I llama_model_loader: - kv  33:            tokenizer.ggml.padding_token_id u32              = 0
0.00.161.693 I llama_model_loader: - kv  34:               tokenizer.ggml.add_bos_token bool             = false
0.00.161.694 I llama_model_loader: - kv  35:                    tokenizer.chat_template str              = {%- if messages[0]['role'] == 'system...
0.00.161.695 I llama_model_loader: - kv  36:            tokenizer.ggml.add_space_prefix bool             = false
0.00.161.695 I llama_model_loader: - kv  37:               general.quantization_version u32              = 2
0.00.161.695 I llama_model_loader: - type  f32:   81 tensors
0.00.161.696 I llama_model_loader: - type q8_0:  282 tensors
0.00.193.067 W llm_load_vocab: special_eos_id is not in special_eog_ids - the tokenizer config may be incorrect
0.00.193.091 I llm_load_vocab: special tokens cache size = 22
0.00.216.974 I llm_load_vocab: token to piece cache size = 0.2826 MB
0.00.216.977 I llm_load_print_meta: format           = GGUF V3 (latest)
0.00.216.978 I llm_load_print_meta: arch             = granite
0.00.216.978 I llm_load_print_meta: vocab type       = BPE
0.00.216.978 I llm_load_print_meta: n_vocab          = 49155
0.00.216.978 I llm_load_print_meta: n_merges         = 48891
0.00.216.978 I llm_load_print_meta: vocab_only       = 0
0.00.216.978 I llm_load_print_meta: n_ctx_train      = 131072
0.00.216.979 I llm_load_print_meta: n_embd           = 4096
0.00.216.979 I llm_load_print_meta: n_layer          = 40
0.00.216.982 I llm_load_print_meta: n_head           = 32
0.00.216.983 I llm_load_print_meta: n_head_kv        = 8
0.00.216.983 I llm_load_print_meta: n_rot            = 128
0.00.216.983 I llm_load_print_meta: n_swa            = 0
0.00.216.983 I llm_load_print_meta: n_embd_head_k    = 128
0.00.216.983 I llm_load_print_meta: n_embd_head_v    = 128
0.00.216.984 I llm_load_print_meta: n_gqa            = 4
0.00.216.985 I llm_load_print_meta: n_embd_k_gqa     = 1024
0.00.216.985 I llm_load_print_meta: n_embd_v_gqa     = 1024
0.00.216.986 I llm_load_print_meta: f_norm_eps       = 0.0e+00
0.00.216.987 I llm_load_print_meta: f_norm_rms_eps   = 1.0e-05
0.00.216.987 I llm_load_print_meta: f_clamp_kqv      = 0.0e+00
0.00.216.987 I llm_load_print_meta: f_max_alibi_bias = 0.0e+00
0.00.216.988 I llm_load_print_meta: f_logit_scale    = 1.6e+01
0.00.216.988 I llm_load_print_meta: n_ff             = 12800
0.00.216.989 I llm_load_print_meta: n_expert         = 0
0.00.216.989 I llm_load_print_meta: n_expert_used    = 0
0.00.216.989 I llm_load_print_meta: causal attn      = 1
0.00.216.989 I llm_load_print_meta: pooling type     = 0
0.00.216.989 I llm_load_print_meta: rope type        = 0
0.00.216.989 I llm_load_print_meta: rope scaling     = linear
0.00.216.990 I llm_load_print_meta: freq_base_train  = 10000000.0
0.00.216.990 I llm_load_print_meta: freq_scale_train = 1
0.00.216.990 I llm_load_print_meta: n_ctx_orig_yarn  = 131072
0.00.216.990 I llm_load_print_meta: rope_finetuned   = unknown
0.00.216.991 I llm_load_print_meta: ssm_d_conv       = 0
0.00.216.991 I llm_load_print_meta: ssm_d_inner      = 0
0.00.216.991 I llm_load_print_meta: ssm_d_state      = 0
0.00.216.991 I llm_load_print_meta: ssm_dt_rank      = 0
0.00.216.991 I llm_load_print_meta: ssm_dt_b_c_rms   = 0
0.00.216.991 I llm_load_print_meta: model type       = 3B
0.00.216.991 I llm_load_print_meta: model ftype      = Q8_0
0.00.216.992 I llm_load_print_meta: model params     = 8.37 B
0.00.216.992 I llm_load_print_meta: model size       = 8.29 GiB (8.50 BPW) 
0.00.216.992 I llm_load_print_meta: general.name     = Models
0.00.216.992 I llm_load_print_meta: BOS token        = 0 '<|end_of_text|>'
0.00.216.993 I llm_load_print_meta: EOS token        = 0 '<|end_of_text|>'
0.00.216.993 I llm_load_print_meta: UNK token        = 0 '<|end_of_text|>'
0.00.216.993 I llm_load_print_meta: PAD token        = 0 '<|end_of_text|>'
0.00.216.993 I llm_load_print_meta: LF token         = 145 'Ä'
0.00.216.993 I llm_load_print_meta: EOG token        = 0 '<|end_of_text|>'
0.00.216.993 I llm_load_print_meta: max token length = 512
0.00.216.994 I llm_load_print_meta: f_embedding_scale = 12.000000
0.00.216.994 I llm_load_print_meta: f_residual_scale  = 0.220000
0.00.216.994 I llm_load_print_meta: f_attention_scale = 0.007812
0.00.255.011 I llm_load_tensors: offloading 40 repeating layers to GPU
0.00.255.013 I llm_load_tensors: offloading output layer to GPU
0.00.255.013 I llm_load_tensors: offloaded 41/41 layers to GPU
0.00.255.016 I llm_load_tensors:    CUDA_Host model buffer size =   204.01 MiB
0.00.255.017 I llm_load_tensors:        CUDA0 model buffer size =  8280.28 MiB
..................................................................................................
0.01.287.921 I llama_new_context_with_model: n_seq_max     = 4
0.01.287.922 I llama_new_context_with_model: n_ctx         = 8192
0.01.287.922 I llama_new_context_with_model: n_ctx_per_seq = 2048
0.01.287.922 I llama_new_context_with_model: n_batch       = 2048
0.01.287.922 I llama_new_context_with_model: n_ubatch      = 512
0.01.287.922 I llama_new_context_with_model: flash_attn    = 0
0.01.287.925 I llama_new_context_with_model: freq_base     = 10000000.0
0.01.287.926 I llama_new_context_with_model: freq_scale    = 1
0.01.287.927 W llama_new_context_with_model: n_ctx_per_seq (2048) < n_ctx_train (131072) -- the full capacity of the model will not be utilized
0.01.287.932 I llama_kv_cache_init: kv_size = 8192, offload = 1, type_k = 'f16', type_v = 'f16', n_layer = 40
0.01.289.475 I llama_kv_cache_init:      CUDA0 KV buffer size =  1280.00 MiB
0.01.289.476 I llama_new_context_with_model: KV self size  = 1280.00 MiB, K (f16):  640.00 MiB, V (f16):  640.00 MiB
0.01.289.891 I llama_new_context_with_model:  CUDA_Host  output buffer size =     0.06 MiB
0.01.296.029 I llama_new_context_with_model:      CUDA0 compute buffer size =   560.00 MiB
0.01.296.031 I llama_new_context_with_model:  CUDA_Host compute buffer size =    24.01 MiB
0.01.296.031 I llama_new_context_with_model: graph nodes  = 1368
0.01.296.032 I llama_new_context_with_model: graph splits = 2
0.01.296.033 I common_init_from_params: setting dry_penalty_last_n to ctx_size = 8192
0.01.296.478 I srv                load_model: chat template, built_in: true, alias: granite, tool call: supported, example:
<|start_of_role|>tools<|end_of_role|>[{"type": "function", "function": {"name": "get_weather", "description": "", "parameters": {"type":"object","properties":{"location":{"type":"string"}}}}},{"type": "function", "function": {"name": "get_temperature", "description": "Return the temperature according to the location.", "parameters": {"type":"object","properties":{"location":{"type":"string"}}}}}]<|end_of_text|>
<|start_of_role|>system<|end_of_role|>You are a helpful assistant. You CAN call tools to assist with the user query. Do not make assumptions about what values to plug into tools. When you can reply with your internal knowledge, reply directly without any tool calls. Otherwise, for each tool call, return <|tool_call|> followed by a JSON list of tool used as follows: <|tool_call|>[{"name": <function-name>, "arguments": <arguments-json-object>}]<|end_of_text|>
<|start_of_role|>user<|end_of_role|>Hello.<|end_of_text|>
<|start_of_role|>assistant<|end_of_role|>Hi there.<|end_of_text|>
<|start_of_role|>user<|end_of_role|>What's the weather like in Paris today?<|end_of_text|>
<|start_of_role|>assistant<|end_of_role|>
0.01.296.480 I srv                      main: initializing server
0.01.296.481 I srv                      init: initializing slots, n_slots = 4
0.01.296.532 I srv                      main: starting server
0.25.036.361 I srv        log_server_request: rid 691254021280 | POST /v1/chat/completions 192.168.50.166:60239
0.25.036.706 I srv oaicompat_completions_req: rid 691254021280 | {"frequency_penalty":0,"logprobs":false,"max_tokens":null,"messages":"[...]","n":1,"parallel_tool_calls":false,"presence_penalty":0,"response_format":{"type":"text"},"seed":null,"stop":null,"stream":true,"stream_options":{"include_usage":true},"temperature":1,"tools":[{"function":{"description":"Get the degrees Celsius temperature value of the given location.","name":"get_temperature","parameters":{"properties":{"location":{"type":"string"}},"required":["location"],"type":"object"}},"type":"function"},{"function":{"description":"Get the weather of the given location.","name":"get_weather","parameters":{"properties":{"location":{"type":"string"}},"required":["location"],"type":"object"}},"type":"function"},{"function":{"description":"Get the city where I am living or working.","name":"where_am_i","parameters":{"properties":{},"required":[],"type":"object"}},"type":"function"}],"top_logprobs":null,"top_p":1}
0.25.704.606 I srv       log_server_response: rid 691254021280 | POST /v1/chat/completions 192.168.50.166:60239 | status 200 | cost 0.67s
```

- chat logs: the tool_call json is incorrect.
```
Q: {"frequency_penalty":0,"logprobs":false,"top_logprobs":null,"max_tokens":null,"n":1,"presence_penalty":0,"response_format":{"type":"text"},"seed":null,"stop":null,"stream":true,"stream_options":{"include_usage":true},"temperature":1,"top_p":1,"tools":[{"type":"function","function":{"name":"get_temperature","description":"Get the degrees Celsius temperature value of the given location.","parameters":{"type":"object","properties":{"location":{"type":"string"}},"required":["location"]}}},{"type":"function","function":{"name":"get_weather","description":"Get the weather of the given location.","parameters":{"type":"object","properties":{"location":{"type":"string"}},"required":["location"]}}},{"type":"function","function":{"name":"where_am_i","description":"Get the city where I am living or working.","parameters":{"type":"object","properties":{},"required":[]}}}],"parallel_tool_calls":false,"messages":[{"role":"system","content":"Today is 2024-12-24.\nYou are a helpful assistant."},{"role":"user","content":"What's the weather like today in ShenZhen?"}]}
A: data: {"choices":[{"delta":{"role":"assistant"},"finish_reason":null,"index":0}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"<tool"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"_call"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":">[{\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"name"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\": \""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"get"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"_weather"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\", \""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"arguments"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\": {\""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"location"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016109,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\": \""},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016110,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"Shen"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016110,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"Zhen"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016110,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":"\"}}"},"finish_reason":null,"index":0,"logprobs":null}],"created":1735016110,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":null}
A: 
A: data: {"choices":[{"delta":{"content":""},"finish_reason":"stop","index":0,"logprobs":null}],"created":1735016110,"id":"chatcmpl-vU9NT8h0NUaJHkmerWlXX9T793RNSMxg","model":"","object":"chat.completion.chunk","usage":{"completion_tokens":27,"prompt_tokens":353,"time_per_output_token_ms":11.005481481481482,"time_to_first_token_ms":27.748,"tokens_per_second":90.86381197248508,"total_tokens":380}}

```

**Comment by thxCode** [comment]
fixed in v0.0.98

**Comment by Finenyaco** [comment]
Verified in v0.0.98
